expr/literals.go

```diff
@@ -220,6 +235,17 @@ func (*NestedLiteral[T]) isRootRef() {}
 func (t *NestedLiteral[T]) String() string {
 	return fmt.Sprintf("%s(%v)", t.Type, t.Value)
 }
+func (t *NestedLiteral[T]) ValueString() string {
+	switch x := any(t.Value).(type) {
+	case ListLiteralValue:
+		var items []string
+		for _, item := range x {
+			items = append(items, item.ValueString())
+		}
+		return fmt.Sprintf("[%s]", strings.Join(items, ", "))
+	}
+	return fmt.Sprintf("%v", t.Value)
```

Comment:
Could we add a test that hits this so the codecov warning goes away?

Reply:
done

Reply:
ok, not done.  Turns out that map literals go through a different path.  I'm not sure how to get it to visit this line.